Orange-OpenSource/Orange-Boosted-Bootstrap
# Prepare v4.4.1

- Firefox ESR
  - IE11
  - Latest Edge, Chrome, Firefox, Safari
  - iOS Safari
  - Chrome & Firefox on Android
- [x] `npm run release-version` to bump version number
  - install [Conventionnal Changelog](https://github.com/conventional-changelog/conventional-changelog) globally
  - `conventional-changelog -p angular -i CHANGELOG.md -s`
  - maintain a ship list … #287 
- [x] **Wait for brand approval or suggestions about current items in navigation**
- [x] implement changes (in components, Boostwtach, examples…)
- [x] re-run `npm run release`
- [x] update ~~Ship List~~ + changelog

Then:
- [ ] merge (on `v4-dev` for now)
- [ ] tag your version
  - attach zip file
  - paste CHANGELOG in the release's description
  - copy `_gh_pages/` top level **files** (`index.html`, `redirects.json`, `sitemap.xml`…) and `_gh_pages/docs/4.4` to the `gh-pages` branch
  - ensure Algolia indexes new release content ([probably requires a PR](https://github.com/algolia/docsearch-configs/pull/811#issuecomment-481250552))
  - if you are already logged in NPM (with a personnal account, for example), [you'd better use a repository scoped `.npmrc` file](https://stackoverflow.com/questions/30114166/how-to-have-multiple-npm-users-set-up-locally)

Close #287 on merge.
